src/Security/Certificate.cs

```diff
@@ -1212,6 +1439,10 @@ public NSData GetKeyExchangeResult (SecKeyAlgorithm algorithm, SecKey publicKey,
 
 #if !NET
 		[Watch (3,0)][TV (10,0)][Mac (10,12)][iOS (10,0)]
+#else
+		[SupportedOSPlatform ("ios10.0")]
+		[SupportedOSPlatform ("tvos10.0")]
+		[SupportedOSPlatform ("macos10.12")]
```

Comment:
^ the last part was *much easier* to review :)

however most should not be needed since `src/MinimumVersions.cs` defines

```csharp
// undecorated API availability falls back to those minimum versions, per platform
[assembly: SupportedOSPlatform ("ios10.0")]
[assembly: SupportedOSPlatform ("tvos10.0")]
[assembly: SupportedOSPlatform ("macos10.14")]
[assembly: SupportedOSPlatform ("maccatalyst13.1")]
[assembly: UnsupportedOSPlatform ("watchos")]
```